Update architecture and storefront setup docs for ACCS and ACO (#746)
commerce-docs/microsite-commerce-storefront: src/content/docs/boilerplate/getting-started.mdx

### Commerce blocks

> The integration of Commerce drop-in components into the Edge Delivery Services architecture of JavaScript blocks and document-based authoring. [Commerce blocks](/get-started/architecture/#boilerplate-and-blocks) are the components that provide the content and layout for commerce pages in the storefront.

Reviewer: Clarify references to "document-based authoring" term used through out.  Since there are three different types of authoring supported.  Not sure if thall-three, and I think the document-based one is the least preferred.   

From AEM documentation (https://www.aem.live/docs/authoring-guide)

Document-based authoring (SharePoint / Google Drive / Document Authoring)
In-context / WYSIWYG authoring (Universal Editor / Storefront Builder)
AEM Sites authoring (Universal Editor on AEM Sites as CMS)

## Understanding the flow

> To understand how documents transform into rendered commerce experiences at runtime, see the [How it works](/get-started/architecture/#how-it-works) section in the architecture guide. This foundational knowledge helps you know where to customize and integrate with the boilerplate.

Reviewer: ```suggestion
To learn how documents become rendered commerce experiences at runtime, see the [How it works](/get-started/architecture/#how-it-works) section in the architecture guide. This foundational knowledge helps you identify how to customize and integrate with the boilerplate.
```
